src/Esfa.Vacancy.UnitTests/Extensions/FixtureExtensions.cs

```diff
@@ -7,24 +8,16 @@ namespace Esfa.Vacancy.UnitTests.Extensions
 {
     public static class FixtureExtensions
     {
-        private const double HoursPerWeekMinimumLength = 16;
-        private const double HoursPerWeekMaximumLength = 48;
-
         public static WageUnit CreateAnyWageUnitOtherThanNotApplicable(this IFixture fixture)
         {
-            return fixture.Create<Generator<WageUnit>>().First(unit => unit != WageUnit.NotApplicable);
-        }
-
-        public static double CreateValidHoursPerWeek(this ISpecimenBuilder fixture)
-        {
-            return fixture.Create<Generator<double>>()
-                .First(d => d >= HoursPerWeekMinimumLength && d <= HoursPerWeekMaximumLength);
+            return fixture.Create<Generator<WageUnit>>()
+                .First(unit => unit != WageUnit.NotApplicable);
         }
 
-        public static double CreateInValidHoursPerWeek(this ISpecimenBuilder fixture)
+        public static DateTime CreateFutureDateTime(this ISpecimenBuilder fixture)
         {
-            return fixture.Create<Generator<double>>()
-                .First(d => d < HoursPerWeekMinimumLength || d > HoursPerWeekMaximumLength);
+            return fixture.Create<Generator<DateTime>>()
```

Comment:
It will be interesting to know how many cycles does the create run before returning a future date.

Reply:
So... it uses [RandomDateTimeSequenceGenerator](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/RandomDateTimeSequenceGenerator.cs) which spits out dates +/-2 years from today. So there's a 50% chance that the next date will be > DateTime.Today.

Reply:
We can't afford this extra cycle potentially. The tests takes ages as it i. Can you just use `DatTime.Now.Add~` to get a future date?